### PR TITLE
Converting Openpype module to Ayon addon

### PR DESCRIPTION
**Pull request by jakubjezek001**
Conversion OpenPype to Ayon following instructions from https://community.ynput.io/t/openpype-to-ayon-core-addon-conversion/1209/3

### CODE REVIEW

File: client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py [hunk 1]
```diff
@@ -62,18 +62,19 @@ def process(self, instance):
...
+                f"'{instance.data['productName']}', skipping "
+                "version creation."
             )
             return
-        
+
         sg_session = context.data["shotgridSession"]
 
         # TODO: Use path template solver to build version code from settings
         anatomy = instance.data.get("anatomyData", {})
```
Comment (iLLiCiTiT) [inline client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py:74]: Note for some future cleanup PR, this variable should not be called `anatomy`.

File: client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py [hunk 2]
```diff
@@ -125,27 +126,29 @@ def process(self, instance):
...
+                # is and its name
                 type_, field_name = sg_field
 
                 data_to_update[field_name] = type_(field_value)
 
-        # Add version objectId to "sg_op_instance_id" so we can keep a link
+        # Add version objectId to "sg_ayon_id" so we can keep a link
         # between both
-        version_entity = instance.data.get("versionEntity", {}).get("_id")
+        version_entity = instance.data.get("versionEntity", {}).get("id")
```
Comment (iLLiCiTiT) [inline client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py:143]: Note for future cleanup PR this variable should not be called `versoin_entity`.

**Review by iLLiCiTiT — APPROVED** [review]
One suggestion for AYON, but approved.

File: client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py (hunk 2 above)
Comment (jakubjezek001) [reply client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py:143]: fixed in f9ddbb37e370bbd1b6c7073afe4ad385858d0fe4

File: client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py (hunk 1 above)
Comment (jakubjezek001) [reply client/ayon_shotgrid/plugins/publish/integrate_shotgrid_version.py:74]: fixed in f9ddbb37e370bbd1b6c7073afe4ad385858d0fe4

**Pull request merged**